### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by GladeOwl**

### CODE REVIEW

File: graph.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from cProfile import label
```
Comment (Demannu) [inline graph.py:1]: Is this needed? It looks like it might've been an accidental auto-import from L14

File: main.py [hunk 2]
```diff
@@ -1,14 +1,16 @@
 import json
 import math
+from operator import inv
 from base import Base
```
Comment (Demannu) [inline main.py:4]: I would choose a different variable name for this, there are a lot of `base` or `Base` used already

File: graph.py (hunk 1 above)
Comment (GladeOwl) [reply graph.py:1]: Seems to be.

File: main.py (hunk 2 above)
Comment (GladeOwl) [reply main.py:4]: Gotcha.

**Pull request merged**